packages/react-components/src/templates/TeamHeader.tsx

```diff
@@ -135,8 +137,26 @@ const TeamHeader: React.FC<TeamProps> = ({
         )}
       </section>
       <TabNav>
-        <TabLink href={aboutHref}>About</TabLink>
-        <TabLink href={outputsHref}>Outputs</TabLink>
+        {tools
```

Comment:
Isn't this way simpler and doesn't need keys if we use `{tools && ...TabLink...}` just on the middle one?

Reply:
Tab Nav will render a li for undefined :(

Reply:
Can we fix that in `TabNav` using
```
  ':empty': {
    display: 'none',
  },
```

Reply:
I'd rather have the element not present than hidden.

Reply:
But this is IMO a bug in `TabNav` and should not be worked around here. Inserting `{false}`, `{null}`, `{<></>}` etc. at arbitrary positions in JSX should not change behavior. Alternative fixes would be more complicated, like iterating over children checking whether they are 'empty'. Also note that `display: none` elements are ignored by screen readers.

Reply:
I already filtered it in tabnav in my amends

Reply:
Did `:empty` not work? The reason why I didn't suggest filtering is exactly because it's a huge PITA, there's infinite cases due to arrays and fragments alone

Reply:
It works in this case and most cases I can think of. Can't we just iterate if it becomes a problem? To test this worked with the CSS solution we would have to do a browser test